po/common/en.po

```diff
@@ -89,15 +90,15 @@ msgstr "Your request to join the organization is pending approval of the organiz
 
 msgctxt "please_email_producers"
 msgid "Please e-mail <a href=\"mailto:producers@openfoodfacts.org\">producers@openfoodfacts.org</a> if you have any question."
-msgstr "Please e-mail <a href=\"mailto:producers@openfoodfacts.org\">producers@openfoodfacts.org</a> if you have any question."
+msgstr "Please e-mail <a href=\"mailto:producers@openfoodfacts.org\"> producers@openfoodfacts.org </a> if you have any question."
 
 msgctxt "if_you_work_for_a_producer"
 msgid "If you work for a producer or brand and will add or complete data for your own products only, you can get access to our completely free Platform for Producers."
 msgstr "If you work for a producer or brand and will add or complete data for your own products only, you can get access to our completely free Platform for Producers."
 
 msgctxt "producers_platform_description_long"
 msgid "The platform for producers allows manufacturers to easily import data and photos for all their products, to mark them as official, and to get free analysis of improvement opportunities for their products."
-msgstr "The platform for producers allows manufacturers to easily import data and photos for all their products, to mark them as official, and to get free analysis of improvement opportunities for their products."
+msgstr ""
```

Comment:
This is a problem, and makes tests fails… (apart from real added translations)

We have a lot of the strings blank in English now ! (and in test it falls back to french !)